lib/pf/condition/network.pm

```diff
@@ -44,11 +40,17 @@ match the last ip to see if it is in defined network
 =cut
 
 sub match {
-    my ($self,$data) = @_;
-    my $last_ip = $data->{last_ip} if exists $data->{last_ip};
-    return defined $last_ip && $self->value->contains(NetAddr::IP->new($last_ip));
+    my ($self,$ip) = @_;
+    return 0 unless defined $ip;
```

Comment:
May the $FALSE be with you?  

--  
LM


On Thursday, September 17, 2015 at 13:15 , Julien Semaan wrote:

> In lib/pf/condition/network.pm (https://github.com/inverse-inc/packetfence/pull/594#discussion_r39772101):
> > @@ -44,11 +40,17 @@ match the last ip to see if it is in defined network > =cut > > sub match { > - my ($self,$data) = @_; > - my $last_ip = $data->{last_ip} if exists $data->{last_ip}; > - return defined $last_ip && $self->value->contains(NetAddr::IP->new($last_ip)); > + my ($self,$ip) = @_; > + return 0 unless defined $ip;  
> use the $FALSE james
>  
>  
> —
> Reply to this email directly or view it on GitHub (https://github.com/inverse-inc/packetfence/pull/594/files#r39772101).
>  
>  
>  
>  
>